build/steps-build-tests.yml

```diff
@@ -0,0 +1,54 @@
+﻿parameters:
+- name: pathToSrc
+  type: string
+  default: ''
+- name: projectName
+  type: string
+  default: ''
+
+steps:
+- template: templates/flutter-install.yml
+
+#-if false
```

Comment:
These conditions were used in the UnoApplicationTemplate to remove pipeline code that isn't relevant in the generated apps. That worked with custom configuration of the dotnet new templating engine. Have you done the same with your PowerShell script? Is that code removed when you generate an app?

Reply:
No because when I asked you before you told me that it was not required for now, but I think it should be easy to include it as a PowerShell script, to just remove those pieces of code when we create a new application using the tool.